dev/deps/spark-deps-hadoop-2.7

```diff
@@ -101,6 +101,7 @@ jakarta.ws.rs-api-2.1.5.jar
 jakarta.xml.bind-api-2.3.2.jar
 janino-3.0.15.jar
 javassist-3.22.0-CR2.jar
+javax.el-3.0.1-b11.jar
```

Comment:
This is a minor technicality, but we'll need to update a few license files if this becomes part of the transitive dependency, as it'll be distributed with Spark:

In `LICENSE-binary`, find:
```
Common Development and Distribution License (CDDL) 1.1
------------------------------------------------------
```
Add an entry to the beginning of this section like:
```
javax.el:javax.el-api  https://javaee.github.io/uel-ri/
```

On a second look, I may need to review whether we need additional licenses in `license-binary/` but that's a larger issue I'd have to go back and remember about, to see if CDDL needs this.

BTW these changes will be needed for the 2.4 backport, but, I think this will back-port cleanly anyway, so may not need the other PR separately.
```